README.md

```diff
@@ -114,180 +114,12 @@ cd ios && pod update && cd ..
-import com.adobe.marketing.mobile.LoggingMode
-import com.adobe.marketing.mobile.MobileCore
-import com.adobe.marketing.mobile.MobileCore.getApplication
-import com.adobe.marketing.mobile.edge.consent.Consent
-import com.adobe.marketing.mobile.edge.identity.Identity
-```
-
-```kotlin
-// MainApplication.kt
- class MainApplication : Application(), ReactApplication {
-   ...
- override fun onCreate() {
-    super.onCreate()
-    
-    MobileCore.setApplication(this);
-    MobileCore.setLogLevel(LoggingMode.DEBUG)
-    MobileCore.configureWithAppID("YOUR-APP-ID");
-    MobileCore.registerExtensions(
-      listOf(
-        Lifecycle.EXTENSION,
-        Edge.EXTENSION,
-        Identity.EXTENSION,
-        Consent.EXTENSION
-      ),
-    ) {
-      Log.d("MainApp", "Adobe Experience Platform Mobile SDK was initialized")
-    }
-
-    SoLoader.init(this, false)
-    if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
-      // If you opted-in for the New Architecture, we load the native entry point for this app.
-      load()
-    }
-    ApplicationLifecycleDispatcher.onApplicationCreate(this)
-  }
-```
-
-```kotlin
-// MainActivity.kt
-
-import android.app.Activity
-import android.app.Application.ActivityLifecycleCallbacks
-import com.adobe.marketing.mobile.MobileCore
-
-// Implementing global lifecycle callbacks
-override fun onCreate(savedInstanceState: Bundle?) {
-    // Set the theme to AppTheme BEFORE onCreate to support
-    // coloring the background, status bar, and navigation bar.
-    // This is required for expo-splash-screen.
-    setTheme(R.style.AppTheme);
-    super.onCreate(null)
-
-    application.registerActivityLifecycleCallbacks(object : ActivityLifecycleCallbacks {
-        override fun onActivityResumed(activity: Activity) {
-            MobileCore.setApplication(application)
-            MobileCore.lifecycleStart(null)
-        }
-
-        override fun onActivityPaused(activity: Activity) {
-            MobileCore.lifecyclePause()
-        }
-
-        // the following methods aren't needed for our lifecycle purposes, but are
-        // required to be implemented by the ActivityLifecycleCallbacks object
-        override fun onActivityCreated(activity: Activity, savedInstanceState: Bundle?) {}
-        override fun onActivityStarted(activity: Activity) {}
-        override fun onActivityStopped(activity: Activity) {}
-        override fun onActivitySaveInstanceState(activity: Activity, outState: Bundle) {}
-        override fun onActivityDestroyed(activity: Activity) {}
-    })
-  }
-```
-
-> For further details on Lifecycle implementation, please refer to the [Lifecycle API documentation](https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#lifecycle).
+> [!NOTE]  
+> Starting from Adobe Experience Platform React native **7.x**,  there is no longer a need to initialize the SDK on the [native platforms](https://github.com/adobe/aepsdk-react-native/tree/docs-v7#initializing), as was required in earlier versions.
```

Comment:
The link here is for the user can refer back to old registering instruction.  The current link doesn't have that info.  
You can create a 6.x branch from Main and point the link to the `initializing` section on  that page.